src/__tests__/shared/components/navbar/Navbar.test.tsx

```diff
@@ -170,13 +170,22 @@ describe('Navbar', () => {
   })
 
   describe('account', () => {
-    it('should render an account link list', () => {
+    it("should render a link with the user's identification", () => {
```

Comment:
There should be a test to make sure that it renders the user's first and last name